src/msc_disk.c

```diff
@@ -31,7 +31,7 @@
 
 #include "msc_disk.h"
 #include <mrubyc.h>
-
+#include "value.h"
```

Comment:
@yswallow 
Do you really need this line?
Please double-check

Reply:
The answer is "No".

I checked comment-outed variables, variable names, function names, etc.

Please see commit "cleanup" 094ce6b.